### DISCUSSION TITLE
Additional message starts with a timestamp

### DISCUSSION THREAD

**Discussion by ary-ber** [Pustur/whatsapp-chat-parser#213] [Q&A]
Hey. I can see that your parser handles the additional messages and merges them with the previous one. What if the additional message is a copy pasted conversation and thus, starts with [m/d/y hh/mm/ss]. It will get attributed to the user in the copied message and not the actual sender. Is there anywhere you are handling this case ? I may have missed it.

**Comment by Pustur** [comment]
Yes, that is an edge case that I didn't know how to avoid, and since I thought that it would rarely happen I decided to leave it in.
If you have suggestions on a fix I'm open to ideas.
